### ISSUE TITLE
GameGuru MAX - Request - More Light Types

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#3317]
According to the developer's blog and this video (see the following link), Wicked Engine has supported various kinds of area lights since at least 2017:

[https://www.youtube.com/watch?v=ZLA47y_u7rM](url)

These types of lights are present in most game systems and would make a fantastic addition to MAX. As you can see in the video (right around the 2:08 minute mark or so), these light shapes can be changed to **_Directional, Point, Spot, Sphere, Disc, Rectangle, and Tube_**. Not only that, but Wicked Engine provides a lot of various controls for lights - **_Energy, Distance, Radius, Width, Height, FOV, Shadow Bias_** and not just Distance and color.

Lighting is essential to making a game look great, particularly indoor-only games. Currently, we are fairly hampered with the two light types that MAX gives us with the limited controls we have over those lights.

Please consider adding these additional light types to MAX.

**Comment by Lemuripest93** [comment]
Yes to this, i'm going to need some different lights for when i'm making a cyberpunk-themed RPG, preferably some befitting a nightclub setting.
